### PR TITLE
model: pytorch: Custom exception when loss function is misspelled

### PR DESCRIPTION
**Pull request by nitesh585**
Solved issue #910.

**Exception message:**
Before change - 'Nonetype' object has no attribute 'function'
After change- Loss function is not valid!

### CODE REVIEW

File: model/pytorch/dffml_model_pytorch/pytorch_base.py [hunk 1]
```diff
@@ -63,6 +63,14 @@ def __post_init__(self):
                 map(self.clstype, self.classifications)
             )
 
+class LossFunctionNotFoundError(Exception):
+    """
+    Exception raised when loss function is misspelled.
+    """
+    def __init__(self, message="Loss function is not valid!"):
```
Comment (pdxjohnny) [inline model/pytorch/dffml_model_pytorch/pytorch_base.py:70]: Not sure if we need this `__init__`. It would probably be sufficient to just pass this error message every time. We should probably say what the spelling they gave was, if possible.
